src/components/submission-common.js

```diff
@@ -107,25 +106,30 @@ export const resolveWdsApp = apps => {
   // look explicitly for a RUNNING app named 'wds-${app.workspaceId}' -- if WDS is healthy and running, there should only be one app RUNNING
   // an app may be in the 'PROVISIONING', 'STOPPED', 'STOPPING', which can still be deemed as an OK state for WDS
   const healthyStates = ['RUNNING', 'PROVISIONING', 'STOPPED', 'STOPPING']
-  const namedApp = apps.filter(app => app.appType === getConfig().wdsAppTypeName && app.appName === `wds-${app.workspaceId}` && healthyStates.includes(app.status))
-  if (namedApp.length === 1) {
-    return namedApp[0]
-  }
 
-  //Failed to find an app with the proper name, look for a RUNNING WDS app
-  const runningWdsApps = apps.filter(app => app.appType === getConfig().wdsAppTypeName && app.status === 'RUNNING')
-  if (runningWdsApps.length > 0) {
-    // Evaluate the earliest-created WDS app
-    runningWdsApps.sort((a, b) => new Date(a.auditInfo.createdDate).valueOf() - new Date(b.auditInfo.createdDate).valueOf())
-    return runningWdsApps[0]
-  }
+  // WDS appType is checked first and takes precedence over CROMWELL apps in the workspace
+  const wdsAppTypes = ['WDS', 'CROMWELL']
```

Comment:
The app type names should be fetched from config as before. It might be good to add a comment in [WM-1803](https://broadworkbench.atlassian.net/browse/WM-1803) mentioning that the `wdsAppTypeName` should be an array instead of single value. FYI @mspector since you are working on WM-1803